members/static/members/css/base.css

```diff
@@ -1,26 +1,26 @@
 ﻿@import url('https://fonts.googleapis.com/css?family=Pirata+One');
 
-ul.navbar-nav {
-    height: 100%;
-    padding: 0;
+.bg-primary {
+    background-color: #25A2CE !important; 
 }
-ul.navbar-nav > li {
-    padding: 0 !important;
+
+.nav-link {
+    font-size: 20px;
+    font-weight: 600;
+}
+
+.navbar-dark .navbar-nav .nav-link {
+    color: white;
+}
+
+body {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
 }
 
-    ul.navbar-nav > li > * {
-        padding: 0.8em 0.5em;
-        display: block;
-        color: white;
-        font-family: 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', Calibri, sans-serif;
-        font-size: 1em;
-    }
-    ul.navbar-nav > li > a.active {
-        border-bottom: 3px solid #ffffff;
-    }
-        ul.navbar-nav > li > a:hover {
-            background-color: #2C8098;
```

Comment:
Menuen skal vøre `#25A2CE` så den ser primær ud. `#2C8098` er en sekundærfarve. Forskellen er hvor lys/mørk de to er

Reply:
Menuen er #25A2CE

Reply:
Og jeg synes footer og men skal være samme farve, jeg laver nok et pull request på en pænere footer senere.